### PR TITLE
[windows][mingwlibs] change installdirectory

### PR DESCRIPTION
**Pull request by Rechi**
## Description
Change the install directory of mingwlibs (FFmpeg and libdvd) to `project/BuildDependencies/mingwlibs/$ARCHITECTURE`
Additionally the dlls no longer get copied to `system` and the copied into the root build directory by cmake, they are now copied by cmake only from their real install directory.

## Motivation and Context
- have mingwlibs in architecture dependant directory
- don't install to lib directory
- don't copy to system folder (architecture dependant files in non architecture dependant directory)

## How Has This Been Tested?
run 32 and 64 bit windows builds

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: lib/libdvd/Makefile.in [hunk 1]
```diff
@@ -1,105 +0,0 @@
-
```
Comment (afedchin) [inline lib/libdvd/Makefile.in]: I'm not sure that this can be dropped, it may be usable on other platforms.

**Review by afedchin — APPROVED** [review]
looks good, except a minor comment
thanks

File: lib/libdvd/Makefile.in (hunk 1 above)
Comment (Rechi) [inline lib/libdvd/Makefile.in]: as commit message says, it's a autotools leftover

Comment (afedchin) [inline lib/libdvd/Makefile.in]: ok then

**Comment by Rechi** [comment]
jenkins build this please

**Review by Paxxi — APPROVED** [review]

**Comment by Rechi** [comment]
also run a jenkins windows 64 bit build successfully

**Pull request merged**